WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -153,11 +154,6 @@ public static void trackSavePostAnalytics(PostModel post, SiteModel site) {
                     AnalyticsUtils.trackWithSiteDetails(AnalyticsTracker.Stat.EDITOR_UPDATED_POST, site, properties);
                 } else {
                     properties.put("word_count", AnalyticsUtils.getWordCount(post.getContent()));
-                    properties.put("editor_source",
```

Comment:
Hmm, did we remove the `editor_source` property on purpose or do we need to migrate how to compute it?

Reply:
yes, I left `editor_source` out for a late pass over it. 
It's quite hard to make it working fine on 100% of cases, but I think i made it working for most of  those  in https://github.com/wordpress-mobile/WordPress-Android/pull/10254/commits/af0d48f6c9b8b58530e3748b7f2ba2faabb535d1

The main problem with this part of the analytics code is that, under some circumstances, we need to access the account store (when the site editor preference is still empty for example) and by using the `user_id` decide if the block editor is the default editor or not. It's a very small % of cases I think, since the migration should have already happened at this point, but it could happen...
On the commit above i've passed the AccountStore to this analytics utils where possible, keeping it null on some cases where introducing it would require lot of refactoring.

Reply:
> keeping it null on some cases where introducing it would require lot of refactoring

Aha, had a look and [this one](https://github.com/wordpress-mobile/WordPress-Android/commit/af0d48f6c9b8b58530e3748b7f2ba2faabb535d1#diff-6f326538ef251c6604082c8eb857f51bR69) is quite trivial (we just need to pass the AccountStore in the constructor and there's only one call site to update).

[The other case](https://github.com/wordpress-mobile/WordPress-Android/commit/af0d48f6c9b8b58530e3748b7f2ba2faabb535d1#diff-07a4b2d77083cfd60941cda4d2aa9a00R265), does indeed require some refactoring.

Reply:
Ok, so let's mark this resolved since we've settled in a Slack convo to not consider the user_id during this migration, resulting in clearing the AccountStore parameter and fix this convo as well.